pkg/antctl/raw/multicluster/create/clusterset.go

```diff
@@ -0,0 +1,156 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package create
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sruntime "k8s.io/apimachinery/pkg/runtime"
+	k8sscheme "k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	mcsscheme "sigs.k8s.io/mcs-api/pkg/client/clientset/versioned/scheme"
+
+	multiclusterv1alpha1 "antrea.io/antrea/multicluster/apis/multicluster/v1alpha1"
+	antreamcscheme "antrea.io/antrea/multicluster/pkg/client/clientset/versioned/scheme"
+	"antrea.io/antrea/pkg/antctl/raw"
+)
+
+type clusterSetOptions struct {
+	leaderCluster          string
+	leaderClusterServer    string
+	leaderClusterPort      string
+	leaderClusterNamespace string
+	memberCluster          []string
+	serviceAccount         []string
+	namespace              string
+	secret                 string
+}
+
+var clusterSetOpt *clusterSetOptions
+
+var clusterSetExamples = strings.Trim(`
+Create a ClusterSet in a Kubernetes cluster
+$ antctl mc create cluster-set <CLUSTER_SET> -n <NAMESPACE> --leader-server <LEADER_SERVER> --member-cluster <CLUSTERS> --service-account <SERVICE_ACCOUNT> --secret <SECRET>
```

Comment:
it would be better to give a real sample here too. you may refer to output of `kubectl get -h`

Reply:
done